### ISSUE TITLE
Public result page

### ISSUE THREAD

**Issue by tjerkkamstra** [alexjustesen/speedtest-tracker#557]
**Is your feature request related to a problem? Please describe.**
Hi,

Thanks for this super nice fork of Henry Whitakkers' docker container. 
Currently it's only possible to view the results of the speedtest after login.
However I use speedtest as a tool so users that connect to my home lab can determine if it's their connection that's causing the slow transfer speeds or wheter it's mine.

**Describe the solution you'd like**
I would love a results page that is accessible without the need of the users logging in.

**Additional context**
If I've overlooked a setting to achieve this, please pardon me and guide me where to look.

**Comment by Sommerbo** [comment]
It isn't in yet, but there is a more comprehensive feature request at #511 The "visitor" role does effectively what you want, if the visitor doesn't have to log in.

**Comment by tjerkkamstra** [comment]
You are right, the only addition indeed not having the visitor role login. @alexjustesen Maybe update the issue @Sommerbo mentioned and close mine?
